src/SUMA/SUMA_Color.c

```diff
@@ -3182,7 +3182,7 @@ SUMA_Boolean SUMA_ScaleToMap_Interactive (   SUMA_OVERLAYS *Sover )
    if (Opt->Clusterize && Opt->RecomputeClust) {
       if (!(nv = (float *)SUMA_calloc(SDSET_VECFILLED(Sover->dset_link),
                                           sizeof(float)))) {
-         SUMA_S_Crit("Failed to alloc11659ate!");
+         SUMA_S_Crit("Failed to allocate!");
```

Comment:
@afni-rickr this typo was not introduced in 478e7b07c241ad45cd9c6bdc1b2defa6facfe294 on purpose e.g. as an expression of emotions or smth like that, right ? ;-)

Reply:
:) Thanks!